code/API_definitions/qod-api.yaml

```diff
@@ -329,18 +402,238 @@ components:
       description: |
         Ports may be specified as a list of ranges or single ports.
     QosProfile:
+      description: |
+        Data type with attributes of a QosProfile
+      type: object
+      properties:
+        name:
+          allOf:
```

Comment:
I've just checked with redocly viewer and this does not work well when the schema in $ref already has a description, as the composed schema will have 2 descriptions and the one in the $ref'ed schema prevails. So if 2 different descriptions are needed for a same $ref'ed schema,  the $ref'ed schema must have no description and the description is moved to the `allOf`